### DISCUSSION TITLE
[A/L] (2021/22) Combinatoria - Esercizio 34

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#210] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152254686-cc5db206-5556-44c5-9941-72edcd9142d1.png)

**Comment by LuginiAndrea** [comment]
### Stringa 05102003
Abbiamo 0<sub>4</sub>, 5<sub>1</sub>, 1<sub>1</sub>, 2<sub>1</sub>, 3<sub>1</sub>
a) Formula delle permutazioni con n oggetti ripetuti -> 8! / (4! * 1!<sup>4</sup>)
b) Gli anagrammi che iniziano con 3 -> 1 * 7! / (4! * 1!<sup>4</sup>), quindi per complemento quelli che non iniziano per 3 -> 8! /  (4! * 1!<sup>4</sup>) - 7! /  (4! * 1!<sup>3</sup>)
c) Risolviamo con PIE.
ab = "05". Anagrammi che iniziano con "05" -> 1 * 1 * 6! / (3! * 1!<sup>3</sup>) = Anagrammi che finiscono con "05". Anagrammi che iniziano e finiscono con "05": 0 in quanto 5 può comparire solo una volta. 
Quindi per PIE quelli che iniziano o finiscono con "05" -> 2 * 6! / (3! * 1!<sup>3</sup>) - 0
